packages/core/src/components/FieldLabel/FieldLabel.tsx

```diff
@@ -43,8 +43,7 @@ const FieldLabel: FC<FieldLabelProps> = forwardRef(
         />
         <label htmlFor={labelFor} ref={ref} className={cx(styles.labelComponentText, labelClassName)}>
           {labelText}
-          {requiredAsterisk && <span className={styles.requiredAsterisk}> *</span>}{" "}
-          {/* Render asterisk based on the prop */}
+          {required && <span className={styles.required}> *</span>}
```

Comment:
I see that you removed the space after the asterisk {" "}, why dont we need it anymore?

Reply:
It has no effect, this is master today:
<img width="451" alt="Screenshot 2024-07-17 at 12 18 15" src="https://github.com/user-attachments/assets/ae6fb249-f98d-4a37-b8a2-eb9902a903e4">
Probably it was added by Prettier. Plus there's no other element on this block